composer.json

```diff
@@ -25,10 +25,11 @@
         }
     },
     "require": {
-        "laravel/framework": "^5.4|^6.0|^7.0|^8.0"
```

Comment:
Didn't you just add support for PHP 5.4 (the middleware)? 🤔

Reply:
Yeah, I did.. 😢